src/org/zaproxy/zap/extension/jython/ExtensionJython.java

```diff
@@ -70,6 +78,43 @@ public void hook(ExtensionHook extensionHook) {
 			PyScriptEngineFactory factory = new PyScriptEngineFactory();
 			this.getExtScript().registerScriptEngineWrapper(new JythonEngineWrapper(factory.getScriptEngine()));
 		}
+		
+		this.jythonOptionsParam = new JythonOptionsParam();
+		extensionHook.addOptionsParamSet(this.jythonOptionsParam);
+		if (View.isInitialised()) {
+			extensionHook.getHookView().addOptionPanel(new JythonOptionsPanel());
+		}
+		
+		extensionHook.addOptionsChangedListener(this);
+	}
+	
+	@Override
+	public void optionsLoaded() {
+		super.optionsLoaded();
+
+		this.modulePath = this.jythonOptionsParam.getModulePath();
+		if (false == Strings.isNullOrEmpty(this.modulePath)) {

```

Comment:
Could this (and other if statements that do the same) be changed to negate the return value instead of comparing to false? That would match the "style" of the other add-ons' and core code.